token.go

```diff
@@ -214,16 +214,61 @@ type TSTInfo struct {
 	Extensions     []pkix.Extension `asn1:"optional,tag:1"`
 }
 
+// Timestamp denots the time at which the timestamp token was created by the TSA
+//
+// Reference: RFC 3161 2.4.2
+type Timestamp struct {
+	// Value is the GenTime of TSTInfo
+	Value time.Time
```

Comment:
```suggestion
	GeneralizedTime time.Time
```

Is it better?

Reply:
Yep, `GeneralizedTime` is indeed a better name.

Reply:
`GeneralizedTime` is not a meaningful name but an ASN.1 syntax. See https://en.wikipedia.org/wiki/GeneralizedTime

Reply:
Going to stick with `Value` then. `Timestamp.Value` sounds reasonable.

Reply:
BTW, `Timestamp` should be a type for general purpose. It is OK to mention RFC3161 in the comments / documentation..